packages/create-svelte/bin.js

```diff
@@ -21,32 +23,33 @@ async function main() {
 	console.log(gray(`\ncreate-svelte version ${version}`));
 	console.log(disclaimer);
 
-	let cwd = process.argv[2] || '.';
-
-	if (cwd === '.') {
-		const opts = await prompts([
-			{
-				type: 'text',
-				name: 'dir',
-				message: 'Where should we create your project?\n  (leave blank to use current directory)'
-			}
-		]);
-
-		if (opts.dir) {
-			cwd = opts.dir;
+	const args = await yargs(hideBin(process.argv)).coerce('types', (arg) =>
+		arg === 'none' ? null : arg
+	).argv;
```

Comment:
this makes it so that `--types=none` ends up as `{ types: null }` internally.

Alternatively we could make the arg pass one-to-one either by making the arg `null` instead of `"none"`, _or_ changing the internal `null` to `"none"`.